### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by dibdas**
Positioning and Floating Elements
[Collaborative Project]
This assignment consists of making a clone of a New York Times article page. 
This project includes :
-  Embedding images
-  Floats and positioning
- Grid CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+                          </ul>
+                        </div>
+                          <div class="logo-container">
+                          <img  src="./images/a.png" href="#newyorktimes" alt="nytimes-logo" class="nytimes-logo pointer">
+                          </div>
+                          <div class="right-nav">
+                            <button type="button" class="nav-button pointer">SUBSCRIBE NOW</button>
+                            <button type="button" class="nav-button pointer">LOG IN</button>
+                          </div>
+                  </navbar>
```
Comment (SpaYco) [inline index.html]: - [x] The Correct tag name is `<nav>`, not `<navbar>`.

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] Fix your linting errors

* [x] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [x]  **The Theory of Inflation** section should be one picture with texts in it

## Optional

- [x] use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
 # positioning-floating-elements
 This assignment consists of making a clone of a New York Times article page.
```
Comment (expjazz) [inline README.md:2]: - You do not need to say that this was an assignment from a school if you do not want to. I believe it is better to say what you showcased in this project, like: 'We cloned the New York Times Pages using grids, flexbox and ...'

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

Hi team :handshake: 

The project looks awesome. Congrats :1st_place_medal: 

Great work implemented what was previously asked :ok_hand: 

You can move forward :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**